src/extension/tab/backend.js

```diff
@@ -0,0 +1,10 @@
+import relay from './tabRelay';
+import { initializeHook } from "./hook";
+import { version as devToolsVersion } from "../manifest.json";
+
+if (document instanceof HTMLDocument) {
```

Comment:
```suggestion
if (typeof document === "object" && document instanceof HTMLDocument) {
```
Is there any chance `document` might not be defined? If so, you should probably use `typeof` to check beforehand.